libdistopia/src/distopia.cpp

```diff
@@ -29,7 +30,15 @@ namespace distopia {
 
             void MinimiseVectors(hn::VFromD<D> &vx,
                                  hn::VFromD<D> &vy,
-                                 hn::VFromD<D> &vz) const {};
+                                 hn::VFromD<D> &vz) const {}
+
+            void MinimalVectors(const hn::VFromD<D> &ix, const hn::VFromD<D> &iy, const hn::VFromD<D> &iz,
+                                const hn::VFromD<D> &jx, const hn::VFromD<D> &jy, const hn::VFromD<D> &jz,
+                                hn::VFromD<D> &ijx, hn::VFromD<D> &ijy, hn::VFromD<D> &ijz) const {
```

Comment:
this signature is kinda fugly as we pass in x, y, z as separate arguments.  We used to have a `VectorTriple` thing which wrapped this.  

Highway has a Tuple3 object which would do the same, have you looked at using this?

Reply:
Seems to be an abstraction of the Vec3 idea, I'll look into it.

Reply:
There is also `SafeFillN ` and `LoadInterleaved4 ` that may be of interest for idx cases.